tests/unit/fixtures/test_catalog.py

```diff
@@ -157,3 +158,23 @@ def test_make_schema() -> None:
         full_name='hive_metastore.abc',
         storage_location='abfss://container1@storage.com',
     )
+
+
+def test_make_volume():
+    ctx, info = call_stateful(make_volume)
+
+    ctx['ws'].volumes.create.assert_called_once()
```

Comment:
can you assert on the actual arguments? :) then we're good to merge

Reply:
PTAL after changes